fe/src/components/PdfDropzone/uploadPdfToApi.ts

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { getToken } from "@/utils/getToken";
+import { BASE_API_URL, SERVICE_DOCUMENT_UPLOAD_ROUTE } from "@/utils/urls";
+import { Dispatch, SetStateAction } from "react";
+import { toast } from "@/lib/reactHotToastExports";
+
+export const uploadPdfToApi = async (
+  file: File,
+  setSuccessUpload: Dispatch<SetStateAction<boolean>>
+) => {
+  const accessToken = await getToken();
+  const reader = new FileReader();
+
+  reader.onload = async () => {
+    // Extract base64 content from data URL
+    const base64Content = reader.result?.toString().split(",")[1];
+    if (base64Content) {
```

Comment:
Now nothing happens if I try to upload empty pdf document. At least toast message should be showed.